src/routes/Board/Board.tsx

```diff
@@ -61,6 +62,7 @@ function Board() {
                       .map((n) => ({...n, authorName: state.users.all.filter((user) => user.id === n.author)[0]?.displayName}))
                       .map((n) => ({...n, votes: state.votes.filter((vote) => vote.note === n.id)}))}
                     votes={state.votes.filter((vote) => vote.note === note.id)}
+                    voteConfiguration={state.voteConfiguration}
```

Comment:
Instead of passing the configuratin through multiple components you could just get it from the store via the app selector in the Votes component

Reply:
Good point. Done